### DISCUSSION TITLE
Showing Video Ads in React Native

### DISCUSSION THREAD

**Discussion by AkshayThakur1995** [microsoft/react-native-windows#11934] [Q&A] [answered]
I am new to React native. I am trying to integrate google IMA sdk in it. I came across an article which showed how to implement it.
Link - https://hackernoon.com/showing-video-ads-in-react-native-part-2-android-including-audio-play ,But I am getting an ERROR  Invariant Violation: requireNativeComponent: "VideoViewManager" was not found in the UIManager.  It is coming from the file VideoViewAndroid.js. Can anyone please help me with it or guide me to the right direction

**Comment by chrisglein** [comment, marked as answer]
Can you provide some more detail? The article discusses `expo-av` up front, but the expo modules do not support Windows as a target. However, the [react-native-video](https://github.com/react-native-video/react-native-video/tree/master/windows) module does support Windows. Or are you attempting to follow the steps to build your own module? Your first best resource for that is going to be [this](https://microsoft.github.io/react-native-windows/docs/native-modules). Without more context as to how far you are on that journey, it's hard to diagnose your error.
